Makefile

```diff
@@ -123,4 +132,36 @@ cluster/prepare/local: cluster/prepare
 	-oc create -f deploy/crds/*.crd.yaml
 	@oc create -f deploy/service_account.yaml
 	@oc create -f deploy/role.yaml
-	@oc create -f deploy/role_binding.yaml
\ No newline at end of file
+	@oc create -f deploy/role_binding.yaml
+
+.PHONY: cluster/prepare/olm
+cluster/prepare/olm: cluster/prepare
+	oc process -p NAMESPACE=$(NAMESPACE) -f deploy/operator-source-template.yml | oc create -f - -n openshift-marketplace
+	oc process -p NAMESPACE=$(NAMESPACE) -f deploy/operator-subscription-template.yml | oc create -f - -n $(NAMESPACE)
+	timeout 1m bash -c 'until oc get crd installations.integreatly.org; do sleep 5; done'
+
+.PHONY: cluster/cleanup/olm
+cluster/cleanup/olm:
+	@-oc delete -f deploy/integreatly-installation-cr.yml --timeout=60s --wait
+	@-oc delete namespace $(NAMESPACE) --timeout=45s --wait
+	@-oc delete catalogsourceconfig.operators.coreos.com/installed-integreatly-operator -n openshift-marketplace
+	@-oc delete operatorsource.operators.coreos.com/integr8ly-operators -n openshift-marketplace
+
+.PHONY: cluster/cleanup/crds
+cluster/cleanup/crds:
+	@-oc delete crd applicationmonitorings.applicationmonitoring.integreatly.org
+	@-oc delete crd blackboxtargets.applicationmonitoring.integreatly.org
+	@-oc delete crd grafanadashboards.integreatly.org
+	@-oc delete crd grafanadatasources.integreatly.org
+	@-oc delete crd grafanas.integreatly.org
+	@-oc delete crd installations.integreatly.org
+	@-oc delete crd webapps.integreatly.org
```

Comment:
Added this target to help cleanup the cluster back to how it would be on a fresh install. This is not currently executed on a cleanup so needs to be run explicitly if you want to cleanup crds.